cmd/create/config.go

```diff
@@ -81,6 +81,17 @@ func NewConfigCmd(tracker *analytics.Tracker) *cobra.Command {
 				},
 				Spec: distroConfig.FuryctlSpec{
 					DistributionVersion: semver.EnsurePrefix(version),
+					ToolsConfiguration: distroConfig.ToolsConfiguration{
+						Terraform: distroConfig.Terraform{
+							State: distroConfig.State{
+								S3: distroConfig.S3{
+									BucketName: "bucket-name",
+									KeyPrefix:  "key-prefix",
+									Region:     "eu-west-1",
```

Comment:
Perhaps we shouldn't hard-code this value here?

Reply:
those dummy values are needed by the minimal config validation, we need to think about it in a different card maybe